plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1057,38 +1075,28 @@ SecRule REQUEST_FILENAME "@endsWith /index.php" \
     nolog,\
     ver:'nextcloud-rule-exclusions-plugin/1.2.0',\
     chain"
-    SecRule ARGS:dbtype "@rx ^(?:my|pg)sql$" \
+    SecRule ARGS:dbtype "@rx ^(?:my|pg|sq)sql$" \
```

Comment:
Is `sqsql` really a valid option? Shouldn't it be `sqlite`?

Reply:
Good catch, I'll add a test for that

Reply:
Fixed. I also found another false positive when working on this.